grants_tagger/bertmesh/train.py

```diff
@@ -0,0 +1,101 @@
+import random
+import math
+import os
+
+from transformers import TFBertModel, AutoModel
+import tensorflow_addons as tfa
+import tensorflow as tf
+import scipy.sparse as sp
+import numpy as np
+import typer
+
+
+class MultiLabelAttention(tf.keras.layers.Layer):
```

Comment:
should this be moved in a file on it's own ?

Reply:
possibly but i might delete the tensorflow train bit anyway. should i keep it? i mean most models are avaiable in pytorch not tensorflow. maybe i leave it and rename it to `train_tf` so the main train is the torch one